### DISCUSSION TITLE
Can't get `lein test` running

### DISCUSSION THREAD

**Discussion by rlhk** [babashka/babashka#1175] [General]
I forked and cloned the repo preparing to submit a PR for improving a test.
However both `lein test` and `./script/test` gives me the following error:
```
Syntax error compiling at (babashka/deps.clj:32:15).
No such var: p/destroy-tree
```
Looking at the source: `src/babashka/deps.clj` it already has `(:require [babashka.process :as p] ...`
What should be done to make it work?

**Comment by borkdude** [comment]
Use java 11

On Sat, 12 Feb 2022 at 08:50, Roger Lam ***@***.***> wrote:

> I forked and cloned the repo preparing to submit a PR for improving a test.
> However both lein test and ./script/test gives me the following error:
>
> Syntax error compiling at (babashka/deps.clj:32:15).
> No such var: p/destroy-tree
>
> Looking at the source: src/babashka/deps.clj it already has (:require
> [babashka.process :as p] ...
> What should be done to make it work?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/babashka/babashka/discussions/1175>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACFSBRC5HTJN44EVESKX63U2YGKVANCNFSM5OGLUSWQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
https://www.michielborkent.nl
https://www.eetvoorjeleven.nu
